modules/gapi/samples/semantic_segmentation.cpp

```diff
@@ -148,29 +146,46 @@ int main(int argc, char *argv[]) {
     // Now build the graph
     cv::GMat in;
     cv::GMat out_blob = cv::gapi::infer<SemSegmNet>(in);
-    cv::GMat out = custom::PostProcessing::on(in, out_blob);
+    cv::GMat post_proc_out = custom::PostProcessing::on(in, out_blob);
+    cv::GMat blending_in = cv::gapi::mulC(in, 0.3, CV_8U);
+    cv::GMat blending_out = cv::gapi::mulC(post_proc_out, 0.7, CV_8U);
+    cv::GMat out = cv::gapi::add(blending_in, blending_out, CV_8U);
```

Comment:
Can this be rewritten with operators? e.g. `*`, `+`, ...

Reply:
Changed to operators.